### DISCUSSION TITLE
Games Missing Audio (Voices) Skyrim, Fallout: New Vegas, etc.

### DISCUSSION THREAD

**Discussion by peachbeach923** [Whisky-App/Whisky#929] [Q&A]
Hello everyone, I've been having an issue running these older games on Steam through Whisky where the audio mostly works, except for the voices - for Fallout: New Vegas this made the game near unplayable as I had to sit through a completely buggy slow mess with no audio or working voices at all, and I just ended up returning the game. Skyrim I've had for a while now and it runs perfectly fine through the initial cutscene, some of the world environment audio plays, but the voices and the music (I think) are just completely absent. 
I've tried looking up fixes, videos and forums and comment sections where people try to fix the issue with the "xaudio2_6" and "xaudio2_7" overrides in the Wine Configuration settings menu, but this ended up breaking my game and I saw people saying to do "Xaudio2_8" instead of 2_7, which made the game run again but the voices are still just missing. Any word of advice on what overrides or settings I can use? I'm not a big computer person so I'm kinda lost and if I have to install a bunch more things I'll probably just give up... So hoping for a simple fix if anyone has one.
Thank you!

**Comment by Dvst1nfvtst** [comment]
https://docs.getwhisky.app/game-support/skyrim-se.html
